<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>481</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>599</width>
       <height>479</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>591</width>
        <height>411</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>待显示图片</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_OpenFolder"/>
   <addaction name="action_OpenFile"/>
   <addaction name="action_Delete"/>
   <addaction name="action_Ergodic"/>
   <addaction name="separator"/>
   <addaction name="action_Enlarge"/>
   <addaction name="action_Narrow"/>
   <addaction name="action_Actual"/>
   <addaction name="action_SuitableWidth"/>
   <addaction name="action_SuitableHeight"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="windowTitle">
    <string>文件目录树</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>节点</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>类型</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <item>
        <property name="text">
         <string>图片</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/folder1.bmp</normaloff>:/icons/icons/folder1.bmp</iconset>
        </property>
        <property name="text">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>分组1</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>图片</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/icons/31.ico</normaloff>:/icons/icons/31.ico</iconset>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>分组2</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>图片</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/icons/31.ico</normaloff>:/icons/icons/31.ico</iconset>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>目录树</string>
    </property>
    <addaction name="action_OpenFolder"/>
    <addaction name="action_OpenFile"/>
    <addaction name="action_Delete"/>
    <addaction name="action_Ergodic"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="action_Enlarge"/>
    <addaction name="action_Narrow"/>
    <addaction name="action_Actual"/>
    <addaction name="action_SuitableWidth"/>
    <addaction name="action_SuitableHeight"/>
   </widget>
   <addaction name="menu_1"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_OpenFolder">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/folder1.bmp</normaloff>:/icons/icons/folder1.bmp</iconset>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="action_OpenFile">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/Documents.ico</normaloff>:/icons/icons/Documents.ico</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/delete1.bmp</normaloff>:/icons/icons/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
  </action>
  <action name="action_Ergodic">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/export1.bmp</normaloff>:/icons/icons/export1.bmp</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="action_Enlarge">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/418.bmp</normaloff>:/icons/icons/418.bmp</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="action_Narrow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/416.bmp</normaloff>:/icons/icons/416.bmp</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="action_Actual">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/414.bmp</normaloff>:/icons/icons/414.bmp</iconset>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
  </action>
  <action name="action_SuitableWidth">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/15.ico</normaloff>:/icons/icons/15.ico</iconset>
   </property>
   <property name="text">
    <string>适合宽度</string>
   </property>
  </action>
  <action name="action_SuitableHeight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/426.bmp</normaloff>:/icons/icons/426.bmp</iconset>
   </property>
   <property name="text">
    <string>适合高度</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/exit.bmp</normaloff>:/icons/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
